### PR TITLE
add reuse metadata and adjust README

### PR DESCRIPTION
**Pull request by Shegox**
**What this PR does / why we need it**:
This PR adds [reuse](https://reuse.software) metadata and mentions the usage of reuse in the README. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
This is required as part of the SAP Open Source Process. 

**Release note**:

```feature user
NONE
```

### CODE REVIEW

File: LICENSES/Apache-2.0.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
+Apache License
```
Comment (Skarlso) [inline LICENSES/Apache-2.0.txt:1]: Why is this file duplicated? We already have it in the LICENSE file.

Comment (Shegox) [reply LICENSES/Apache-2.0.txt:1]: [Reuse as a tool requires that all licenses are stored in the LICENSES subfolder](https://reuse.software/spec/). 

- `./LICENSE` pretty much indicates the overall license of a project and is used to indicate the license to platforms like GitHub. 
- `./.reuse/dep5` together with `LICENSES` indicates file-specific licenses and copyright information. This is only really relevant if you have "copied" third-party source inside the project e.g. when you vendor dependencies into your project. This spec is used by e.g. https://clearlydefined.io/ to determine all licenses used in a repository.

Ideally you only want one format and the `reuse` approach would be the more precise one, but then you would lose license indication on platform like GitHub. Therefore we opt to have both.

Comment (Skarlso) [reply LICENSES/Apache-2.0.txt:1]: Ok

**Review by Skarlso — APPROVED** [review]

**Comment by Skarlso** [comment]
@Shegox Please sign your commit.

**Comment by Skarlso** [comment]
Thank you :))

**Pull request merged**